lib/interface/cli/commands/trigger/type/get.cmd.js

```diff
@@ -0,0 +1,50 @@
+const debug = require('debug')('codefresh:cli:get:trigger-types');
+const Command = require('../../../Command');
+const _ = require('lodash');
+const { trigger } = require('../../../../../logic').api;
+const { specifyOutputForSingle, specifyOutputForArray } = require('../../../helpers/get');
+const getRoot = require('../../root/get.cmd');
+
+const command = new Command({
+    command: 'trigger-types [type] [kind]',
```

Comment:
lets model the filters: type and kind as options and not arguments

- should be able to get a trigger-type by id/name

`codefresh get trigger-types`
`codefresh get trigger-type ID/NAME`
`codefresh get trigger-types --type registry`
`codefresh get trigger-types --kind cfcr`

Reply:
`kind` does not make sense without `type`. And not all types have kinds. E.g. `registry` type can have `dockerhub`, `ecr`, cfcr` and other kinds that represent different Docker registries, while `cron` type does not have any `kind`.

Reply:
that's ok.
still they should be options to filter when you get.
you can define that an option is required with another option if needed

Reply:
Will add task to JIRA backlog